src/website/views/components/tabs/_examples.pug

```diff
@@ -1161,7 +1275,7 @@ h2 Examples
     | To preserve it, specify a target property on the link.
   .example.-mb--3
     .-p--3.-pb--4
-      ul#example-default-link-behavior.chi-tabs(role="tablist" aria-label="example-default-link-behavior")
+      ul#example-default-link-behavior.chi-tabs(role="tablist" aria-label="Tabs")
         li.-active
```

Comment:
This example is missing web-component example and codesnippet

Reply:
Done